src/main/java/sirius/biz/storage/layer3/ExtractArchiveJob.java

```diff
@@ -156,6 +156,11 @@ private void handleExtractedFile(ExtractedFile extractedFile,
                                   .format());
 
         Watch watch = Watch.start();
+
+        if (checkEmptyFile(extractedFile, process, watch)) {
```

Comment:
```suggestion
        if (extractedFile.size() == 0) {
            logEmptyFile(...);
            return;
        }
```

This way one immediatelly knows what is checked and what isn't